examples/cpustat.lua

```diff
@@ -0,0 +1,29 @@
+--
+-- SPDX-FileCopyrightText: (c) 2025 Enderson Maia <endersonmaia@gmail.com>
+-- SPDX-License-Identifier: MIT OR GPL-2.0-only
+--
+-- Simple example demonstrating linux.cpustat() usage
+
+local linux = require("linux")
+
+-- Get number of CPUs
+local num_cpus = linux.nproc()
+print(string.format("System has %d online CPUs\n", num_cpus))
+
+-- Get stats for each CPU
+for i = 0, num_cpus - 1 do
```

Comment:
I'm not sure it's the right way to do it.. I think we can have gaps.. for instance, have 3 CPU online, out of 4 existent.. I would take a look at foreach_online_cpu macro in the kernel source to make sure..

Reply:
I'll send a fixup where I already introduce a new module called `luaprocstat`

Not sure about the name yet, but was an attempt to narrow the scope, we can discuss it later.

I'm just in doubt if I create a new PR, or keep with this one.

And if keeping this one, if I should squash so we can start a ne round of reviews, 'cause I think it's starting to get messy.

You tell me.